### ISSUE TITLE
Problem with "$_GET" video 40

### ISSUE THREAD

**Issue by jmlab** [thedigicraft/Atom.CMS#91]
Before everything else, I think your tutorials are the best of the web. Very well explained :)
I've done a project before, where I used your tutorial and it worked perfectly. However, a new project, forced me to go a step ahead.

Differences:
Tree / recursive menu.

I still can not enter the form data in the database, and I get an error when I try to echo the database data in the form with the "$ _GET ['id']."
Can you help me?

The code:
=============================================================================
Insert query:
![insert_query](https://cloud.githubusercontent.com/assets/3187774/5714293/9ccf7674-9abe-11e4-82b8-991cd45efa21.jpg)

Tree/Recursive Menu
![treemenu](https://cloud.githubusercontent.com/assets/3187774/5714301/adde5ebc-9abe-11e4-888e-767e08c19e8f.jpg)

Form
![form](https://cloud.githubusercontent.com/assets/3187774/5714304/b9c94e76-9abe-11e4-87c1-6faf8bd13da3.jpg)

Table
![dbtable](https://cloud.githubusercontent.com/assets/3187774/5714308/c55b08b0-9abe-11e4-9463-11439a9cdc1e.jpg)

=============================================================================

Thank you very much
JP (Portugal)

**Comment by creptor** [comment]
if you can't do it, there has to be an error. please port it. if you can't get it in your config.php place: `error_reporting(1);` (please the actual error so it's easier to find the problem)
Images are hard to work with, please use http://pastebin.com/
to get data from the database into the form the developer used the query (query.php) please post that file.(it was also combined with the data.php)

**Comment by thedigicraft** [comment]
Yes as creptor says; please post the error and place the code in here so I can better debug it.
